### ISSUE TITLE
Are we ready to rename SwiftGRPCNIO to "GRPC"?

### ISSUE THREAD

**Issue by timburks** [grpc/grpc-swift#454]
### New Issue Checklist

- [x] Updated grpc-swift to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/grpc/grpc-swift/blob/master/CONTRIBUTING.md)
- [x] I searched for [existing GitHub issues](https://github.com/grpc/grpc-swift/issues)(especially, there are many [resolved issues related to SSL](https://github.com/grpc/grpc-swift/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+ssl))

### Issue Description

I think these imports (below) from a sample application would read much better if we renamed "SwiftGRPCNIO" to "GRPC". This renaming would also support deprecating SwiftGRPC, which could be renamed to something like "GRPCClassic" until EOL. 

```
import Dispatch
import Foundation
import SwiftGRPCNIO
import OAuth2
import NIO
import NIOHTTP1
import NIOSSL
```

**Comment by glbrntt** [comment]
Almost!

We have a few other things we need to do before we can deprecate SwiftGRPC. I plan on starting a discussion thread so we can figure out what we're missing from SwiftGRPCNIO, how we support SwiftGRPC and for how long and also what we can do to help users migrate from SwiftGRPC/GRPCClassic to GRPC.

**Comment by MrMage** [comment]
Also, keep in mind that the NIO version requires users to be on Swift 5 (I am not yet, for example), and we might also want to have a NIOTransportServices variant before we declare it "ready". I would suggest that we should keep a non-NIO branch under maintenance at least until the end of 2019. (Shouldn't be much work, though.)

**Comment by dcow** [comment]
@glbrntt did that discussion thread every happen? I'm curious about making the switch myself and want to triage/prep.

**Comment by glbrntt** [comment]
@dcow not really, we had a few informal discussions about naming and support but nothing concrete yet, I just created #526, however.

FWIW: the NIO version has now been renamed to `GRPC` and has a pre-release tag: [1.0.0-alpha.1](https://github.com/grpc/grpc-swift/releases/tag/1.0.0-alpha.1) if you wanted to give it a go!

**Comment by MrMage** [comment]
Closing this in favor of #526.
